crates/bindings/src/table.rs

```diff
@@ -388,6 +388,30 @@ impl<Tbl: Table, Col: Index + Column<Table = Tbl>> UniqueColumn<Tbl, Col::ColTyp
         let buf = IterBuf::take();
         update::<Tbl>(Col::index_id(), new_row, buf)
     }
+
+    /// Attempts to insert `new_row` into the table. If it exists, deletes the row with the same value in the unique column.
```

Comment:
I think this doc comment is not super clear. It seems to imply: Attempt to insert a new row. If successful, exit. If failure, instead delete the existing row, then exit without (re-)inserting the new row.

I would instead say something like:

```suggestion
    /// Insert `new_row` into the table,
    /// first checking for an existing row with a matching value in the unique column
    /// and deleting it if present.
```

Please use this or similar language both here and in the comment on the panicking version.